packages/health-check/bin/cli

```diff
@@ -16,6 +16,16 @@ require("yargs/yargs")(process.argv.slice(2))
       console.log(await getAuthCookie(true));
     }
   )
+  .command(
+    "apikey",
+    "Get test user's API key",
+    () => {},
+    async () => {
+      const { getAPIKey } = require("../src/utils");
+
+      console.log(await getAPIKey());
+    }
+  )
```

Comment:
I don't think we need this method if we've decided to set test api key in env variables, other services should use the env variable directory instead.